ui/components/multichain-accounts/multichain-address-row/multichain-address-row.tsx

```diff
@@ -67,7 +76,7 @@ export const MultichainAddressRow = ({
   };
 
   const handleQrClick = () => {
-    console.log('QR code clicked for address:', address);
+    onQrClick?.(address, networkName, networkImageSrc);
```

Comment:
Updating handleQrClick with callback to provide necessary props for the QR code modal to render correctly